### PR TITLE
Support Microsoft.Bcl.AsyncInterfaces

### PR DESCRIPTION
**Pull request by pengweiqhca**
Support netstandard 2.0 and net461 of https://github.com/benaadams/Ben.Demystifier/pull/125

### CODE REVIEW

File: test/Ben.Demystifier.Test/Ben.Demystifier.Test.csproj [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\src\Ben.Demystifier\key.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
```
Comment (bruno-garcia) [inline test/Ben.Demystifier.Test/Ben.Demystifier.Test.csproj]: What about going with 5.0.0? instead? It's 10 months old:
https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/5.0.0

Comment (pengweiqhca) [reply test/Ben.Demystifier.Test/Ben.Demystifier.Test.csproj]: Updated

File: test/Ben.Demystifier.Test/Ben.Demystifier.Test.csproj [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\src\Ben.Demystifier\key.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
```
Comment (WeihanLi) [inline test/Ben.Demystifier.Test/Ben.Demystifier.Test.csproj]: No need for .NET Core 3.0 and above, maybe we could add a condition like follows:

``` xml
<PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" Condition="'$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netcoreapp2.1'" />
```

**Comment by benaadams** [comment]
Could you rebase?

**Comment by pengweiqhca** [comment]
I fixed the conflict

**Review by bruno-garcia — APPROVED** [review]
LGTM :)

**Review by benaadams — APPROVED** [review]

**Comment by benaadams** [comment]
Thank you

**Pull request merged**